solr/core/src/resources/EnvToSyspropMappings.properties

```diff
@@ -13,10 +15,15 @@ SOLR_TIMEZONE=user.timezone
 SOLR_TIP=solr.install.dir
 SOLR_TIP_SYM=solr.install.symDir
 SOLR_WAIT_FOR_ZK=waitForZk
+ZK_CLIENT_TIMEOUT=zkClientTimeout
+ZK_CREATE_CHROOT=createZkChroot
+ZK_HOST=zkHost
 
 # These should not be mapped to system properties
```

Comment:
why should these not be mapped to system properties?  wouldn't you want to be able to set some of these that way??

Reply:
The ones I marked as no map are either SSL props that must anyway be set in script because they are read by Jetty (?), or variables that are temp variables in the script (SOLR_OPTS etc) that have no real use as sysProps, but are still exported since we export everything `SOLR_*`

Reply:
The client SSL options (`SOLR_SSL_CLIENT_*`, `SOLR_SSL_CHECK_PEER_NAME`) are not read by the Jetty Server, they are read instead by the SolrClients. They should be mapped.

You are missing `SOLR_SSL_CLIENT_HOSTNAME_VERIFICATION` for the non-mapped options. This is a server option, so it will be read by the jetty config on startup.

Reply:
Ah, so as long as `EnvUtils` is in solr core, it cannot be used in SolrJ. Perhaps `EnvUtils` should be moved to SolrJ?

Reply:
So I added `SOLR_SSL_CLIENT_HOSTNAME_VERIFICATION` to non-mapped, and added the CLIENT mappings back, have a look.